code/go/0chain.net/smartcontract/dbs/event/transaction.go

```diff
@@ -50,6 +50,16 @@ func (edb *EventDb) GetTransactionByClientId(clientID string, limit Pagination)
 	return tr, res.Error
 }
 
+// GetTransactionByToClientId searches for transaction by toClientID
+func (edb *EventDb) GetTransactionByToClientId(toClientID string, limit Pagination) ([]Transaction, error) {
+	var tr []Transaction
+	res := edb.Store.Get().Model(Transaction{}).Where(Transaction{ToClientId: toClientID}).Offset(limit.Offset).Limit(limit.Limit).Order(clause.OrderByColumn{
```

Comment:
we need index for it

Reply:
Added indexes for all missing queried columns :)